databricks_cli/pipelines/cli.py

```diff
@@ -187,6 +187,49 @@ def reset_cli(api_client, pipeline_id):
     click.echo("Reset triggered for pipeline {}".format(pipeline_id))
 
 
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Starts the execution of a delta pipeline run')
+@click.option('--pipeline-id', default=None, type=PipelineIdClickType(),
+              help=PipelineIdClickType.help)
+@debug_option
+@profile_option
+@pipelines_exception_eater
+@provide_api_client
+def run_cli(api_client, pipeline_id):
+    """
+    Starts the execution of a delta pipelines run by starting the cluster and processing data.
+
+    Usage:
+
+    databricks pipelines run --pipeline-id 1234
+    """
+    _validate_pipeline_id(pipeline_id)
+        PipelinesApi(api_client).run(pipeline_id)
+    click.echo("Run triggered for pipeline {}".format(pipeline_id))
```

Comment:
Fix the indenting here?

I don't know Python well enough, but I thought this was an error.

Reply:
Yeah that is strange, all the tests passed though. Going to look into it.

Reply:
Actually it looks like the build on this PR did fail

Reply:
Yeah you're right, I think i just messed up when running tox then